.github/workflows/ci_docs.yml

```diff
@@ -3,16 +3,27 @@ on:
   push:
     # Run only on changes on these files
     paths:
-      - 'lib/**.nim'
+      - 'compiler/docgen.nim'
+      - 'compiler/renderverbatim.nim'
       - 'doc/**.rst'
-      - 'doc/nimdoc.css'
+      - 'doc/nimdoc.css' # xxx is it worth duplicating just because of this entry which rarely changes?
+      - 'lib/**.nim'
+      - 'nimdoc/testproject/expected/testproject.html'
```

Comment:
The tester is run as part of the standard pipeline: https://github.com/nim-lang/Nim/blob/devel/koch.nim#L559.

You don't need to specify it here

Reply:
no the point is the following: if changes were made that required you to change 'nimdoc/testproject/expected/testproject.html', it's likely that you either fixed something in docgen or changed the way rendering works, in which case you want to regenerate docs to reflect those changes